index.html

```diff
@@ -0,0 +1,256 @@
+          </div>
+          <div>
+            Never use a reference manager again.
+            Provide a DOI, ISBN, PubMed ID, url, etc., and let Manubot generate and number citations for you.
+            See your manuscript update right away as you make changes. 
+            Test and deploy automatically with <a href="https://travis-ci.org/">continuous integration</a>.
+          </div>
+        </div>
+      </div>
+      <div class="row flex_wrap_reverse">
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-hand-point-up fa-lg"></i> Beautiful and interactive
+          </div>
+          <div>
+            Static is out, dynamic is in.
+            Publish your paper with the contemporary look and interactive features people have come to love and expect from modern webpages: tooltips, public annotations, table of contents, figure viewer, and more.
+          </div>
+        </div>
+        <div class="col_image"></div>
+      </div>
+    </section>
+    <section>
+      <div class="row flex_wrap">
+        <div class="col_image"></div>
+        <div class="col_text">
+          <div class="col_header">
+            <i class="fas fa-lock-open fa-lg"></i> Free and open-source
+          </div>
+          <div>
+            There's no need to pay to produce brilliant manuscripts.
+            Manubot is 100% free and open-source.
+            Its tools are developed and supported by a community of responsive and passionate software developers and researchers.
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="special_section">
+      <div class="section_header">
+        Made With Manubot
+      </div>
+      <div class="row flex_wrap">
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+        <div class="example_image"></div>
+      </div>
+      <div class="row flex_wrap">
+        <a href="" class="button_link">
+          <i class="fas fa-book-open fa-lg"></i> Full Catalog
+        </a>
+      </div>
+    </section>
+    <section>
+      <div class="section_header">
+        Our Philosophy
+      </div>
+      <div class="row">
+        Manuscripts are the primary basis of scholarly communication, yet the process of creating them has not kept in step with the advancement of technology or the evolving best-practices of science.<br />
+        <br />
+        Traditionally, authoring of manuscripts takes place in a very closed manner.
+        While most scientific material is eventually distributed to some kind of public medium, the process of developing and reviewing the material is usually done in private, contained to a handful of individuals.
```

Comment:
A key point here is the delays of the traditional publishing process discussed in https://www.nature.com/news/does-it-take-too-long-to-publish-research-1.19320

Reply:
I'll put this in. Additionally, if you have relevant links to the other points i list in the paragraph, that would be great to show people.

Reply:
cc @agitter and reminder to @dhimmel 

Links to papers or any kind of references (even if they're informal) would be good to have for the "our philosophy" section. Helps back up our claims of what is wrong with the current system and/or how we fix those problems.

Reply:
There are some references that may be relevant:
- http://blogs.nature.com/naturejobs/2017/06/01/techblog-c-titus-brown-predicting-the-paper-of-the-future/
- https://doi.org/10.22541/au.148769949.92783646
- https://www.nature.com/news/scientific-writing-the-online-cooperative-1.16039

They aren't perfectly aligned with the Manubot philosophy but will give you some options to consider.